packages/wonder-blocks-dropdown/src/components/dropdown-core.tsx

```diff
@@ -964,6 +976,10 @@ class DropdownCore extends React.Component<Props, State> {
                             maxDropdownHeight,
                         ),
                     ]}
+                    aria-invalid={role === "listbox" ? ariaInvalid : undefined}
+                    aria-required={
+                        role === "listbox" ? ariaRequired : undefined
+                    }
```

Comment:
**suggestion:** It would be good adding a comment explaining why we have to add the condition here.